src/Components/Artwork/Details.tsx

```diff
@@ -203,12 +223,22 @@ export const Details: React.FC<DetailsProps> = ({
     saveOnlyToDefaultList,
   } = useArtworkGridContext()
 
+  const { showCollectorSignalBadge } = useCollectorSignalsContext()
+
   const isP1Artist = rest?.artwork.artist?.targetSupply?.isP1
   const isHighDemand =
     Number((rest?.artwork.marketPriceInsights?.demandRank || 0) * 10) >= 9
 
   const showHighDemandInfo = !!isP1Artist && isHighDemand && showHighDemandIcon
 
+  // TODO: replace randomization with real signals and business logic
+  const hasCollectorSignalBadge: boolean = useMemo(
+    () => !!showCollectorSignalBadge && Math.random() < 0.2,
```

Comment:
Just for demo-ing the visual. We'll need to implement the desired data fetching and logic, in a separate PR.